generate_stablehlo.py

```diff
@@ -0,0 +1,278 @@
+# LD_LIBRARY_PATH=~/miniconda3/envs/torch310/lib/:/usr/lib/x86_64-linux-gnu/ PJRT_DEVICE=CPU python xla/generate_stablehlo.py
+import shutil
+import os
+import re
+import numpy as np
+import torch
+from torch import nn
+import torch_xla
+from torch_xla.core import xla_model as xm
+import jax
```

Comment:
please leave a comment clarifying jax is a temp poc artifact that we plan to replace - to avoid confusion.